antiope-s3/src/Antiope/S3.hs

```diff
@@ -163,7 +163,7 @@ deleteFiles b ks = do
   let dObjs = delete' & dObjects .~ (objectIdentifier <$> ks)
   resp <- AWS.send (deleteObjects b dObjs)
   unless (List.null $ resp ^. drsErrors) $
-    fail (resp ^. drsErrors & show)
+    error (resp ^. drsErrors & show)
```

Comment:
`MonadAWS` is `MonadThrow`, would it be more correct to use `throwM` here instead of `error`?

Reply:
Because the old implementation pretty much just called `error` then this change is fine for fixing compilation for the new compiler.

We can modify it to use `throwM` but because this is a change in behaviour, it should deserver a more significant version bump and can be done subsequent to a release for `ghc-8.8.1`.

Reply:
I just think that using `fail` was a mistake in a first place.